### DISCUSSION TITLE
Almost perfect

### DISCUSSION THREAD

**Discussion by hcarpentier** [mdbootstrap/TW-Elements#1792] [Kind words & general feedback]
Hi everyone, thanks for all the work done, it's an amazing tool ! The only thing I don't quite get until now is how to deal with responsivity using TE. Indeed, basic usage of Tailwind responsivity doesn't seem to work anymore with TE. It's like the CSS of TE overwrites the one of Tailwind for some styling and for the media queries which is very limiting when dealing with responsivity. Do you have a similar behavior ? Anyone has a solution ?

**Comment by juujisai** [comment]
Hi, the default tailwind breakpoints should work properly. We have added a new `xs` breakpoint that can make elements with `container` class look a little bit different than the default tailwind one. We have already a task to check that more in detail.

Is this the issue you are experiencing?

**Comment by hcarpentier** [comment]
Hi, I'm not sure it's due to xs breakpoint. It's more about TE overriding the Tailwind rules. 
I'm importing TE with : <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/tw-elements/dist/css/tw-elements.min.css" />
For example, h1 tags have a default formatting coming from TE so I need to redefine properly the style of all the h1 if I'm using TE.
Is it normal ? Is there any reset.css file coming TE to prevent this behavior ?

Thanks a lot!

**Reply by juujisai** [reply]
Default classes for the `h1` and `code` tags are going to be removed in the next release. The only thing that will be changed is the default line height for `p` tags (1.6 instead of 1.5).
